crates/iota-storage/src/http_key_value_store.rs

```diff
@@ -109,7 +106,6 @@ fn key_to_path_elements(key: &Key) -> IotaResult<(String, &'static str)> {
             encoded_tagged_key(&TaggedKey::CheckpointSequenceNumber(*seq)),
             "cs",
         )),
-        Key::CheckpointContentsByDigest(digest) => Ok((encode_digest(digest), "cc")),
```

Comment:
Oh did they forget to remove that upstream? 😄

Reply:
It seems so, yes.